platform/linux-generic/pktio/dpdk.c

```diff
@@ -14,6 +14,9 @@
 #include <ctype.h>
 #include <unistd.h>
 #include <math.h>
+#if !defined(__arm__) || defined(__aarch64__)
+#include <numa.h>
+#endif
```

Comment:
How about adding:
```
#else
#define numa_num_configured_nodes() 1
#endif
```
to eliminate the need for the other `#if` below? Also, are we using `!__arm__` as a synonym for x86_64? If I'm running on PPC or MIPS is `<numa.h>` available?

Reply:
I'd also prefer dummy `numa_num_configured_nodes()` implementation for the platforms where it is not supported.

Reply:
Configure test seems perfectly fine here.

Reply:
Makes sense, will update this.

Reply:
Would the following be OK for everyone?

```
#if RTE_EAL_NUMA_AWARE_HUGEPAGES
#include <numa.h>
#else
#define numa_num_configured_nodes() 1
#endif
```

DPDK build system will take care of detecting numa awareness.

Reply:
That is ok for me.

Reply:
@matiaselo fine with me also